packages/python-sdk/e2b/exceptions.py

```diff
@@ -30,10 +30,10 @@ class TimeoutException(SandboxException):
     """
     Raised when a timeout occurs.
 
-    The [unavailable] exception type is caused by sandbox timeout.\n
-    The [canceled] exception type is caused by exceeding request timeout.\n
-    The [deadline_exceeded] exception type is caused by exceeding the timeout for process, watch, etc.\n
-    The [unknown] exception type is sometimes caused by the sandbox timeout when the request is not processed correctly.\n
+    The `unavailable` exception type is caused by sandbox timeout.\n
```

Comment:
There is similar comment in TS SDK like this so we might want to change it there too.
Overall we need to improve how we communicate the errors, so I'm not sure if we should be touching this right now.

Reply:
Ok i made this change based on feedback, but we should keep it consistent so i will edit the ts comment